managedtenants/utils/ocm.py

```diff
@@ -29,6 +29,8 @@ def retry_hook(exception):
 
 class OcmCli:
     API = "https://api.stage.openshift.com"
+    ADDON_MGMT_API_URL_PREFIX = "/api/clusters_mgmt/v1/addons"
+    ADDON_MGMT_NEW_API_URL_PREFIX = "/api/addons_mgmt/v1/addons"
```

Comment:
```suggestion
    CLUSTERS_SERVICE_ADDON_MGMT_API_URL_PREFIX = "/api/clusters_mgmt/v1/addons"
    ADDONS_SERVICE_ADDON_MGMT_API_URL_PREFIX = "/api/addons_mgmt/v1/addons"
```

wdyt

Reply:
I have shorten the Variable name as follows 
```
    CS_ADDON_MGMT_API_URL_PREFIX = "/api/clusters_mgmt/v1/addons"
    AS_ADDON_MGMT_API_URL_PREFIX = "/api/addons_mgmt/v1/addons"
```

Reply:
perfect, thanks!